paddle/cuda/src/hl_cuda_cublas.cc

```diff
@@ -162,6 +169,65 @@ void hl_matrix_transpose(real *A_d, real *C_d, int dimM, int dimN) {
   hl_matrix_transpose(A_d, C_d, dimM, dimN, dimN, dimM);
 }
 
+void hl_matrix_inverse(real *A_d, real *C_d, int dimN, int lda, int ldc) {
```

Comment:
cublasSgetrfBatched is used to calculate a number of small-sized matrices. 
So, it is best to comment that the performance of this interface may be poor.